### ISSUE TITLE
应该与各自图中的真实框的类别号中验证pred_class_logits所指的实际类别号在各自图中的真实框的类别号中是否存在

### ISSUE THREAD

**Issue by ghost** [bubbliiiing/mask-rcnn-keras#51]
老师好：
关于刚才所提的Mask R-CNN的mrcnn_training.py的mrcnn_class_loss_graph()的多分类损失计算问题，在验证pred_class_logits所指的实际类别号在各自图中的真实框的类别号中是否存在时，
其代码为：pred_active = tf.gather(active_class_ids[0], pred_class_ids) 为何所有pred_class_logits多分类输出所指的实际类别号都在第一幅图中的真实框的类别号中(即active_class_ids[0])验证，应该与各自图中的真实框的类别号中验证呀？谢谢
